cosmos/precompile/auth/auth_test.go

```diff
@@ -283,12 +283,12 @@ var _ = Describe("Address Precompile", func() {
 				ctx,
 				evm,
 				common.Address{},
-				big.NewInt(0),
+				new(big.Int),
 				false,
 				granter,
 				grantee,
 				sdkCoinsToEvmCoins(limit),
-				big.NewInt(0),
+				new(big.Int),
```

Comment:
we should do separate changes, like this refactor, in diff PRs

Reply:
for future yeah, but yolo for now.